### PR TITLE
🎉 Initial Commit

### PR DESCRIPTION
**Pull request by StatArman**

### CODE REVIEW

File: lab4/src/main/java/search/Main.java
Comment (iramh) [inline lab4/src/main/java/search/Main.java:1]: The user does not get any information if the word you are searching for is not in the file.

**Review by iramh — COMMENTED** [review]
Check the comment

**Review by iramh — APPROVED** [review]
review comment

File: lab4/src/main/java/search/Main.java
Comment (StatArman) [reply lab4/src/main/java/search/Main.java:1]: This is intended behaviour, if the output would be piped to another program, there shouldn't be any other printouts

One could add an error output, but that's mostly quality of life stuff

**Pull request merged**